content/docs/release-notes/release-notes-1.11.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Release 1.11
+description: 'cert-manager release notes: cert-manager 1.11'
+---
+
+> ⚠️ This release is not yet ready for production.
+> The first stable version is due to be published in January 2023.
+> The latest **pre-release** version is [`v1.11.0-beta.0`](https://github.com/cert-manager/cert-manager/releases/tag/v1.11.0-beta.0).
+
+## Community
+
+Thanks again to all open-source contributors with commits in this release, including:
+
+- [@cmcga1125](https://github.com/cmcga1125)
+- [@karlschriek](https://github.com/karlschriek)
+- [@lvyanru8200](https://github.com/lvyanru8200)
+- [@mmontes11](https://github.com/mmontes11)
+- [@pinkfloydx33](https://github.com/pinkfloydx33)
+- [@sathyanarays](https://github.com/sathyanarays)
+- [@weisdd](https://github.com/weisdd)
+- [@yann-soubeyrand](https://github.com/yann-soubeyrand)
```

Comment:
suggestion: This is missing at least `SpectralHiss` from the release notes in this PR.

Looking through commits since the v1.10 feature freeze on master using [this list](https://github.com/cert-manager/cert-manager/compare/v1.10.0-beta.0...master) I also see:

- `joycebrum` (https://github.com/cert-manager/cert-manager/pull/5544)
- `Git-Jiro` (https://github.com/cert-manager/cert-manager/pull/5478)
- `thib-mary` (https://github.com/cert-manager/cert-manager/pull/5555)
- `yk` (https://github.com/cert-manager/cert-manager/pull/5628)
- `RomanenkoDenys`  (https://github.com/cert-manager/cert-manager/pull/5637)
- `lucacome` (https://github.com/cert-manager/cert-manager/pull/5629)

Reply:
(As part of this, I noticed that most of the PRs since the release of 1.10 were assigned to the 1.10 milestone rather than the 1.11. I've fixed this manually for them all now I think!)

Reply:
(Gonna add these people as part of a commit adding release notes for 1.11.0-beta.1)